pytest_repo_health/fixtures/github.py

```diff
@@ -40,14 +40,16 @@ async def github_repo(git_origin_url, github_client, loop):  # pylint: disable=r
     """
     if git_origin_url is None:
         # There isn't an origin for this repository or directory
-        return None
+        raise Exception("There isn't an origin for this repository or directory")
+
     match = re.search(URL_PATTERN, git_origin_url)
     if match is None:
         # The origin isn't hosted on GitHub (might be GitLab, Bitbucket, etc.)
-        return None
+        raise Exception("The origin isn't hosted on GitHub")
```

Comment:
The problem with this is that it causes any checks based on this fixture to fail, whereas what we want is:

1. The check can tell that it didn't get a repo object, and respond accordingly (that's why it gets `None` currently)
2. The information about why we didn't get a repo object is stored or logged somwhere where it can be found if that wasn't the expected result